client/app/components/calendar/event/DeleteConfirmation.jsx

```diff
@@ -1,26 +1,55 @@
 import React from 'react';
 import { Modal, Button } from 'react-bootstrap';
+import EventSchema from 'schemas/EventSchema';
+import { eventTimeString } from 'helpers/DateHelper';
+import moment from 'moment';
 
-const DeleteConfirmation = ({ onHide, show, cancelHandler, confirmHandler }) => (
-  <Modal bsSize="small"
-         show={show}
+const TIME_FORMAT = 'HH:mm';
+const DATE_FORMAT = 'Mo MMM YYYY';
+
+const DeleteConfirmation = ({ onHide, show, onCancel, onConfirm, event }) => (
+  <Modal show={show}
          onHide={onHide}
          className="delete-confirmation-modal">
-    <Modal.Header className="text-center">
-      Are you sure?
+    <Modal.Header className="text-center" closeButton>
+      Are you sure you want to delete this event?
     </Modal.Header>
-    <Modal.Footer className="confirmation-footer">
-      <Button onClick={cancelHandler}>Cancel</Button>
-      <Button bsStyle="danger" onClick={confirmHandler}>Delete</Button>
+    <Modal.Body className="delete-confirmation-modal modal-body">
+      <h1 className="summary">{event.summary}</h1>
+      <p>
+        <em>on </em>
+        <strong>{moment(event.start.date_time).format(DATE_FORMAT)} </strong>
+        ({eventTimeString(event, TIME_FORMAT)})
+      </p>
+      <p>
+        <em>in </em>
+        {event.conference_room.title}
+      </p>
+      <p>
+        <em>by </em>
+        {event.creator.email}
+      </p>
```

Comment:
Could you just render `<Event />` here instead of repeating this? You can customize it with `dateFormat` and `timeFormat` props